src/Bellatrix.Web/services/DevToolsService.cs

```diff
@@ -28,6 +30,99 @@ public DevToolsService(IWebDriver wrappedDriver)
 
     public DevToolsSessionDomains DevToolsSessionDomains { get; set; }
     public DevToolsSession DevToolsSession { get; set; }
+    public List<NetworkRequestSentEventArgs> RequestsHistory { get; set; }
+    public List<NetworkResponseReceivedEventArgs> ResponsesHistory { get; set; }
+
+    public void StartMonitoringBiDiApi()
```

Comment:
maybe this should be renamed to StartNetworkTrafficMonitoring no need to mention that you use BiDiApi considering that it is in this class

Reply:
Method renamed